gum/arch-arm64/gumarm64writer.h

```diff
@@ -98,10 +101,20 @@ void gum_arm64_writer_put_sub_reg_reg_imm (GumArm64Writer * self,
 void gum_arm64_writer_put_nop (GumArm64Writer * self);
 void gum_arm64_writer_put_brk_imm (GumArm64Writer * self, guint16 imm);
 
+void gum_arm64_writer_put_push_all_registers (GumArm64Writer * self);
+void gum_arm64_writer_put_push_all_q_registers (GumArm64Writer * self);
+
```

Comment:
Perhaps we could drop this empty line? They kind of belong to the same group I'd say.